boards/arm/nucleo_g474re/nucleo_g474re.dts

```diff
@@ -147,10 +147,10 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		/* Set 2Kb of storage at the end of the 128Kb of flash */
-		storage_partition: partition@1f800 {
+		/* Set 4Kb of storage at the end of the 128Kb of flash */
+		storage_partition: partition@1f000 {
 			label = "storage";
-			reg = <0x0001f800 0x00000800>;
+			reg = <0x0001f000 0x00001000>;
```

Comment:
Would probably be good to use `DT_SIZE_K(4)` for better readability.

Reply:
Done, thanks.